src/test/java/seedu/programmer/storage/JsonProgrammerErrorStorageTest.java

```diff
@@ -96,10 +96,10 @@ public void saveProgrammerError_nullProgrammerError_throwsNullPointerException()
     /**
      * Saves {@code ProgrammerError} at the specified {@code filePath}.
```

Comment:
Since parameter has changed, should change the javadoc comment too 😄

Reply:
Yeap, makes sense!